.github/workflows/release.yml

```diff
@@ -71,3 +93,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.TOKEN_ACCESS }}
         with:
           args: 'install/build/*.tar.gz.MD5'
+      - name: Upload Seed (Database)
+        uses: skx/github-action-publish-binaries@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOKEN_ACCESS }}
+        with:
+          args: 'adempiere/Adempiere/data/ExpDat.dmp'
```

Comment:
Would it be possible to rename the ExpDate.dmp as Adempiere_pg.jar and include it in the artifact zip file?

Reply:
Hi @mckayERP I change it by Export reference and export as Adempiere_pg.jar
For add ir as articact practically I should will change all current build because right now the build generate a Zip and tar and it has not splitted as build and generate zip

I think that is better if this PR can be acepted and make other only for change the build and reduce processing time also